src/TestBootstrap.lua

```diff
@@ -35,38 +35,49 @@ local function getPath(module, root)
 	return path
 end
 
---[[
-	Find all the ModuleScripts in this tree that are tests.
-]]
-function TestBootstrap:getModules(root)
-	local modules = {}
+local function toStringPath(tablePath)
+	local stringPath = ""
+	local first = true
+	for _, element in ipairs(tablePath) do
```

Comment:
Is this not just `table.concat(tablePath, " ")`?

Reply:
Nope, order is reversed